### PR TITLE
Skip hubble exception check due to language & Add cleanup script

### PR DESCRIPTION
**Pull request by Pengzna**
In revision 1.3.0, some message are written in Chinese, which will fail in some test encoding in English.
So we need to ignore it

&& Add cleanup script

### CODE REVIEW

**Comment by Pengzna** [comment]
@imbajin PTAL~

**Comment by imbajin** [comment]
THX, rebase/merge master for the CI?

File: src/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+set -x # see detailed output
```
Comment (imbajin) [inline src/cleanup.sh:1]:
```suggestion
set -ex # see detailed output
```

File: src/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+kill -9 `jps | grep HugeGraph | awk '{print $1}' `
+rm -rf ../graph
+rm -rf ./config/dataset
```
Comment (imbajin) [inline src/cleanup.sh:4]:
```suggestion
rm -rf ./config/dataset

```

**Review by imbajin — APPROVED** [review]

**Pull request merged**